crates/bevy_render/src/view/view.wgsl

```diff
@@ -64,3 +64,204 @@ struct View {
     mip_bias: f32,
     frame_count: u32,
 };
+
+/// World space:
+/// +y is up
+
+/// View space:
+/// -z is forward, +x is right, +y is up
+/// Forward is from the camera position into the scene.
+/// (0.0, 0.0, -1.0) is linear distance of 1.0 in front of the camera's view relative to the camera's rotation
+/// (0.0, 1.0, 0.0) is linear distance of 1.0 above the camera's view relative to the camera's rotation
+
+/// NDC (normalized device coordinate):
+/// https://www.w3.org/TR/webgpu/#coordinate-systems
+/// (-1.0, -1.0) in NDC is located at the bottom-left corner of NDC
+/// (1.0, 1.0) in NDC is located at the top-right corner of NDC
+/// Z is depth where: 
+///    1.0 is near clipping plane
+///    Perspective projection: 0.0 is inf far away
+///    Orthographic projection: 0.0 is far clipping plane
+
+/// UV space:
+/// 0.0, 0.0 is the top left
+/// 1.0, 1.0 is the bottom right
+
+
+// -----------------
+// TO WORLD --------
+// -----------------
+
+/// Convert a view space position to world space
+fn position_view_to_world(view_pos: vec3<f32>, world_from_view: mat4x4<f32>) -> vec3<f32> {
+    let world_pos = world_from_view * vec4(view_pos, 1.0);
+    return world_pos.xyz;
```

Comment:
note: most of these functions are the same thing with different names. "augment point with w = 1.0, mul, return xyz" or same thing but with w = 0.0 for directions. Do we want to unify them?

Reply:
Not sure it's worth it. These functions are mostly so people don't need to understand exactly what the correct maths should be, I guess. Convenience. But extracting functions for... transform_position() and transform_direction() seems unnecessary to me.